sceptre/cli/delete.py

```diff
@@ -30,6 +30,7 @@ def delete_command(ctx, path, change_set_name, yes):
     """
     context = SceptreContext(
         command_path=path,
+        command_params=ctx.params,
```

Comment:
Is disable rollback an option on delete as well?

Reply:
no, it's not.